.changes/unreleased/Features-20220318-085756.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Features
+body: Support custom names for generic tests
```

Comment:
Is it just generic tests or all tests?

Reply:
Singular tests take their name from the file in which they're defined (`tests/my_custom_test.sql`), just like models — so they've always required custom names